sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/core/MultiTenantDBCosmosFactory.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.data.cosmos.core;
+
+import com.azure.cosmos.CosmosAsyncClient;
+import com.azure.spring.data.cosmos.CosmosFactory;
+import com.azure.spring.data.cosmos.config.CosmosConfig;
+import com.azure.spring.data.cosmos.core.convert.MappingCosmosConverter;
+import org.springframework.data.auditing.IsNewAwareAuditingHandler;
+
+/**
+ * Template class for cosmos db
+ */
+public class MultiTenantDBCosmosFactory extends CosmosFactory {
```

Comment:
Would be great to add this example in the readme, and then later have a complete sample link in the readme.

Reply:
Done.